records/validation_test.go

```diff
@@ -35,15 +37,17 @@ func TestValidateResolvers(t *testing.T) {
 		{[]string{}, true},
 		{[]string{""}, false},
 		{[]string{"", ""}, false},
-		{[]string{"a"}, false},
-		{[]string{"a", "b"}, false},
+		{[]string{"localhost"}, true},
+		{[]string{"example.com"}, true},
+		{[]string{"example.com", "localhost"}, true},
+		{[]string{"localhost", "localhost"}, false},
 		{[]string{"1.2.3.4"}, true},
-		{[]string{"1.2.3.4.5"}, false},
+//		{[]string{"1.2.3.4.5"}, false},
```

Comment:
try to parse as an IP (there's a golang lib func for this); if it's not an
IP then validate hostname syntax (we'll probably have to write a func for
this).

On Tue, Sep 8, 2015 at 4:58 PM, Karl Isenberg <notifications@github.com>
wrote:

> In records/validation_test.go
> <https://github.com/mesosphere/mesos-dns/pull/251#discussion_r38979335>:
>
> >  		{[]string{"1.2.3.4"}, true},
> > -		{[]string{"1.2.3.4.5"}, false},
> > +//		{[]string{"1.2.3.4.5"}, false},
>
> How do you validate both 1.2.3.4.5 AND hostnames? I don't know of any way
> to do it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/251/files#r38979335>.
>